src/main/java/com/epam/grid/engine/provider/host/slurm/SlurmHostProvider.java

```diff
@@ -0,0 +1,71 @@
+package com.epam.grid.engine.provider.host.slurm;
```

Comment:
for all new classes please add a license

Reply:
done